### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by zWhdmB5T** [TeamNewPipe/NewPipe#10703] [Questions]
Thank you!

Thank you for your project and software!

Thank you for maintenance in the past and in the future!

Kind regards and season's greetings!

**Comment by OldGuyInTheClub** [comment]
I'd like to agree with this sentiment. Newpipe is terrific!

**Comment by Gametrek** [comment]
I enjoy Newpipe and try to get my parents and neighbors to install, use, and update the application when they can. Problematically they do not listen. Just continue to use regular Youtube and waste time looking at ____ , which could save them some time. %100 all the time on any device I come across. 

Only thing I do not like lately is how various material is being censored where it was viewable. Especially news sources, or current events. It is like anything that exposes anybody seems to blocked, but the content is still there.
